app/src/main/java/lt/vilnius/tvarkau/AboutActivity.java

```diff
@@ -0,0 +1,115 @@
+package lt.vilnius.tvarkau;
+
+import android.content.ActivityNotFoundException;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.widget.Toolbar;
+import android.widget.TextView;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import lt.vilnius.tvarkau.utils.GlobalConsts;
+
+public class AboutActivity extends BaseActivity {
+
+    @BindView(R.id.toolbar)
+    Toolbar toolbar;
+
+    @BindView(R.id.version_code)
+    TextView versionCode;
+
+    @BindView(R.id.municipality_phone)
+    TextView municipalityPhone;
+
+    @BindView(R.id.municipality_email)
+    TextView municipalityEmail;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_about);
+        ButterKnife.bind(this);
+
+        setSupportActionBar(toolbar);
+        if (getSupportActionBar() != null) {
+            getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        }
+
+        versionCode.setText(BuildConfig.VERSION_NAME);
+        municipalityPhone.setText(GlobalConsts.VILNIUS_MUNICIPALITY_PHONE);
+        municipalityEmail.setText(GlobalConsts.VILNIUS_MUNICIPALITY_EMAIL);
+    }
+
+    @OnClick(R.id.rate_app)
+    protected void onRateAppClick() {
+        // TODO testPackage is kept for testing; afterwards change testPackage to this.getPackageName();
+        String testPackageName = "uk.co.es4b.tvarkau_vilniu";
```

Comment:
This TODO looks tricky. Maybe put it somewhere in global config class, so it would be replaced later in that one location?